config/runner.yml

```diff
@@ -22,6 +22,7 @@ drupal:
   root: "build"
   root_absolute: ~
   base_url: "http://127.0.0.1:8888"
+  settings_local_file: 'settings.local.php'
```

Comment:
Drupal core already uses `settings.local.php` shall we maybe rename this as below?

```
  settings_override_file: 'settings.override.php'
```